src/iris/api.py

```diff
@@ -5388,6 +5388,29 @@ def on_delete(self, req, resp, username, application):
         resp.status = HTTP_204
 
 
+class InternalApplicationsAuth(object):
```

Comment:
Since we moved off of python 2.7, we don't need to inherit from object anymore.

Reply:
good point, we should do a cleanup of any leftover python 2.7 cruft